WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -1580,6 +1580,12 @@ public Cursor getMediaFile(String blogId, String mediaId) {
         return db.rawQuery("SELECT * FROM " + MEDIA_TABLE + " WHERE blogId=? AND mediaId=?", new String[]{blogId, mediaId});
     }
 
+    public Cursor getMediaFileByVideoPressId(String blogId, String videoId) {
+        String shortcode = "[wpvideo " + videoId + "]";
```

Comment:
This routine along with `getVideoPressVideoPosterFromURL()` below doesn't check whether the passed strings are `null`.